doc/release_notes.rst

```diff
@@ -13,6 +13,8 @@ Upcoming Release
 
 **New Features and major Changes**
 
+* Identify DC lines but temporary transform them back into AC `PR #348 <https://github.com/pypsa-meets-africa/pypsa-africa/pull/348>`
```

Comment:
Missing the ending "__" to make that an url

Reply:
Corrected. Thanks! :)